data/main.ts

```diff
@@ -102,6 +102,7 @@ const setupInfrastructure = async (): Promise<void> => {
         "openai_api_key",
         "google_books_api_key",
         "google_oauth_client_secret",
+        "encryption_key",
```

Comment:
cc: @mdroidian - if you're ever creating employees locally and need to ssh into the box, you'll need an `ENCRYPTION_KEY` env var locally